spec/game_logic_test_spec.rb

```diff
@@ -0,0 +1,78 @@
+# spec/game_logic_test_spec.rbc
+
+require '../lib/board'
```

Comment:
`require_relative '../lib/board'`
`require_relative '../lib/game_logic'`

[REQUIRED] You need to use the `require_relative` syntax instead to prevent errors.